src/controllers/controller.bar.js

```diff
@@ -162,6 +162,10 @@ module.exports = DatasetController.extend({
 			label: me.chart.data.labels[index]
 		};
 
+		if (helpers.isArray(dataset.data[index])) {
```

Comment:
what functionality does this provide? Is it needed with the new border width changes?

Reply:
`isArray` means that it's the floating bar case. The idea is that we want to draw all the borders by default for the floating bar case. Normally we skip the border that's on the axis, but if no borders are on the axis then it makes sense to draw them all. The new border width change simplified this code quite a bit by adding support for skipping no borders